src/strings/en-us.json

```diff
@@ -1716,5 +1716,9 @@
     "LabelTonemappingMode": "Tone mapping mode",
     "TonemappingModeHelp": "Select the tone mapping mode. If you experience blown out highlights try switching to the RGB mode.",
     "Unknown": "Unknown",
-    "AllowAv1Encoding": "Allow encoding in AV1 format"
+    "AllowAv1Encoding": "Allow encoding in AV1 format",
+    "AiTranslated": "AI Translated",
```

Comment:
Just a small tip for future changes, if you add translations alphabetically you don't encounter merge conflicts as often. :wink: